static_src/components/users.jsx

```diff
@@ -94,7 +94,7 @@ export default class Users extends React.Component {
   }
 
   get resourceGuid() {
-    const resourceGuid = this.state.current === ORG_NAME ?
+    const resourceGuid = this.state.currentType === ORG_NAME ?
```

Comment:
This might be a good opportunity to add a unit test for this JSX component. Or a functional test. In the past we didn't do unit tests for components, but we have one or two examples now that do shallow rendering with enzyme.